docs/ssh-tutorial.md

```diff
@@ -0,0 +1,138 @@
+# Tutorial
+
+Create a reverse proxy to allow SSH access to a computer
+
+Scenario: You want to allow SSH access to a computer that doesn't have a public IP, is inside a private network or behind
+a firewall
+
+
+
+## Setup your exit node
+
+#### Using inletsctl
+You will need to have an account and API key with one of the [supported providers](https://github.com/inlets/inletsctl#featuresbacklog), 
+we are using Digital Ocean in this example. So download your access key to a file called `do-access-token` and put it in
+your home directory.
+
+
+First of all, download `inletsctl` to your local machine
+```sh
+# Install Inletsctl to /usr/local/bin
+curl -sLSf https://inletsctl.inlets.dev | sudo sh
+
+# Create an exit node
+
+inletsctl create --access-token-file ~/do-access-token --remote-tcp 127.0.0.1
+```
+This should then create an exit node, which will forward all TCP traffic to through the client to the `remote-tcp` address
```

Comment:
Did this work? I seemed to need the IP of eth0

Reply:
You are correct :man_facepalming: wrote this in 2 halves and must have not tested the full flow.

Changed and tested full flow now